ibm/mas_devops/roles/suite_install/defaults/main.yml

```diff
@@ -116,3 +116,6 @@ mas_img_pull_policy: "{{ lookup('env', 'MAS_IMG_PULL_POLICY') }}"
 # Custom Labels
 # -----------------------------------------------------------------------------
 custom_labels: "{{ lookup('env', 'CUSTOM_LABELS') | default(None, true) | string | ibm.mas_devops.string2dict() }}"
+
+# Enable IPv6?
+enable_ipv6: "{{ lookup('env', 'ENABLE_IPV6') | default('false', true) }}"
```

Comment:
this needs an entry in https://github.com/ibm-mas/ansible-devops/blob/4de7f6f21eb2244872f919a736d9a9750cba2429/ibm/mas_devops/roles/suite_install/README.md and needs to state what it does, and that it is currently only supported in internal devit fyre clusters.

The variable should also have the `| bool` filter at the end as it a boolean. Any checks can then check it as a boolean rather than a "true" string.